node/src/BaseClient.ts

```diff
@@ -2917,10 +2935,14 @@ export class BaseClient {
      * console.log(result); // Output: Set {} - An empty set is returned since the key does not exist.
      * ```
      */
-    public async spopCount(key: string, count: number): Promise<Set<string>> {
-        return this.createWritePromise<string[]>(createSPop(key, count)).then(
-            (spop) => new Set<string>(spop),
-        );
+    public async spopCount(
+        key: GlideString,
+        count: number,
+        decoder?: Decoder,
+    ): Promise<Set<GlideString>> {
+        return this.createWritePromise<GlideString[]>(createSPop(key, count), {
+            decoder: decoder,
```

Comment:
```suggestion
            decoder,
```

Reply:
All these are now updated to the new DecoderOption